src/prefixes.json

```diff
@@ -6,6 +6,7 @@
         "CMO": "http://purl.obolibrary.org/obo/CMO_",
         "COB": "http://purl.obolibrary.org/obo/COB_",
         "EFO": "http://purl.obolibrary.org/obo/EFO_",
+        "FOO": "https://purl.ihccglobal.org/FOO_",
```

Comment:
Was this just for testing, or is it used somewhere?

Reply:
Sorry, that was a test. I removed it.